CMakeLists.txt

```diff
@@ -604,7 +604,7 @@ set(RECASTNAVIGATION_DEMO OFF CACHE BOOL "Do not build RecastDemo")
 set(RECASTNAVIGATION_STATIC ON CACHE BOOL "Build recastnavigation static libraries")
 set(RECASTNAVIGATION_TESTS OFF CACHE BOOL "Do not build recastnavigation tests")
 
-add_subdirectory (extern/recastnavigation)
+add_subdirectory (extern/recastnavigation EXCLUDE_FROM_ALL)
```

Comment:
What does this do specifically? EXCLUDE_FROM_ALL?

Reply:
As far as I understand, it excludes subdirectory contents from the default build & install targets. As OpenMW target references it explicitly, it would still be built but not installed.

https://cmake.org/cmake/help/v3.13/prop_dir/EXCLUDE_FROM_ALL.html
https://knzl.at/cmake-exclude-from-all/

Reply:
Noice. :D